src/pages/cookiebeleid/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { NextPage } from 'next';
+import Script from 'next/script';
+import React from 'react';
+
+import styles from './cookie-policy.module.scss';
+
+const CookiePolicy: NextPage = () => {
+	return (
+		<div className={styles['cookie-policy__wrapper']}>
```

Comment:
Use a (probably Sticky) layout and `p-`-prefix for page-level classes

Reply:
The cookie policy page should look something like this?
https://onderwijs-qas.hetarchief.be/cookiebeleid

How would a sticky layout work for that?